wamr-compiler/main.c

```diff
@@ -153,6 +154,7 @@ main(int argc, char *argv[])
     option.enable_aux_stack_check = true;
     option.enable_bulk_memory = true;
     option.enable_ref_types = true;
+    option.enable_gc = false;
```

Comment:
Should also add related code to control the feature for LLVM JIT:
https://github.com/bytecodealliance/wasm-micro-runtime/blob/main/core/iwasm/interpreter/wasm_loader.c#L3053
After it, add:
```C
#if WASM_ENABLE_GC != 0 || WASM_ENABLE_GC_BINARYEN != 0
    option.enable_gc = true;
    option.enable_ref_types = false;
#endif
```